src/utils/embedded/device-nonce/device-nonce.utils.ts

```diff
@@ -36,6 +36,8 @@ export function generateDeviceNonce(): DeviceNonce {
 }
 
 export async function storeDeviceNonce(deviceNonce: DeviceNonce): Promise<DeviceNonce> {
+  console.trace("Updating deviceNonce =", deviceNonce);
```

Comment:
When playing around with Odysee and the Playground on different tabs, I sometimes get different deviceNonce values, so I want to see if it's because we are overriding it for some reason.